packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -112,14 +112,14 @@ class TabController extends ChangeNotifier {
        _animationController = animationController;
 
 
-  /// Creates a new [TabController] with `index`, `previousIndex`, and `length`
-  /// if they are non-null.
-  ///
-  /// This will reuse the existing [_animationController].
-  ///
-  /// This is useful for [DefaultTabController], for example when
-  /// [DefaultTabController.length] is updated, this method is called so that a
-  /// new [TabController] is created without having to create a new [AnimationController].
+  // Creates a new [TabController] with `index`, `previousIndex`, and `length`
+  // if they are non-null.
+  //
+  // This method will reuse the existing [_animationController].
+  //
+  // This is useful for [DefaultTabController], for example when
```

Comment:
This wasn't the target of your change, but I think the explanation for this can be broken down since it's a very long sentence. Maybe something along the lines of: 

"For [DefaultTabController], when [DefaultTabController.length] is updated, this method is called to create a new [TabController] without creating a new [AnimationController]."

Reply:
There was also a discussion on using `///` for this comment in https://github.com/flutter/flutter/pull/30884#discussion_r278334998. Ultimately, they decided to use `///` since it is public-quality private documentation.